modules/migration-creating-migration-plan-cam.adoc

```diff
@@ -27,37 +27,48 @@ The *Plan name* can contain up to 253 lower-case alphanumeric characters (`a-z,
 . Select the projects to be migrated and click *Next*.
 . Select *Copy* or *Move* for the persistent volume (PV):
 
-* *Copy* copies the data in a source cluster's PV to the replication repository and then restores it on a newly created PV, with similar characteristics, in the target cluster.
+. Select a *Source cluster*, a *Target cluster*, and a *Repository*, and click *Next*.
+. In the *Namespaces* screen, select the projects to be migrated and click *Next*.
```

Comment:
Suggestion: add colons after *Copy* and *Move*

Reply:
No. *Copy* and *Move* are the subjects of the verb "copies" and make grammatical sense. There is no reason to turn these sentences into fragments.